src/WebJobs.Script/Description/Workers/WorkerFunctionInvoker.cs

```diff
@@ -80,23 +87,28 @@ protected override async Task<object> InvokeCore(object[] parameters, FunctionIn
                 Logger = context.Logger
             };
 
-            _logger.LogDebug($"Sending invocation id:{invocationId}");
+            string invocationId = context.ExecutionContext.InvocationId.ToString();
+            _logger.LogTrace($"Sending invocation id:{invocationId}");
```

Comment:
For OOP invocations, we were writing like 6 log statements on every request. Making them Trace statements so they're not logged by default, but will be in "diagnostic" mode.

Reply:
Well, Pragna has already reviewed and signed off. I've retained the logs at trace level, so if we could make changes in the future if needed.

Reply:
Logs coming from `RpcWorkerChannel.cs` will be enough for us to RCA/CRI analysis.  ACK on moving other invocation logs to TraceLevel.